pubspec.lock

```diff
@@ -123,6 +123,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.0.2"
+  rxdart:
```

Comment:
what's this be? I've never touched this file. what it do?

Reply:
rxdart is an extension of Stream -> allows you to do streams easier and more understandably
https://medium.com/flutter-community/why-use-rxdart-and-how-we-can-use-with-bloc-pattern-in-flutter-a64ca2c7c52d

but in this case its just used for its `Obersvable<T>`